src/workspaces/dashboard/AzureStorageDetails.test.ts

```diff
@@ -1,11 +1,32 @@
 import { screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import * as clipboard from 'clipboard-polyfill/text';
```

Comment:
is this a normal pattern to use * for import?

Reply:
No, not in our "regular" code (though it sometimes still is done for utils files). But it is necessary here for mocking.